### DISCUSSION TITLE
Service Apache Cannot Start

### DISCUSSION THREAD

**Discussion by PPPPPP130** [leokhoa/laragon#749] [Q&A]
![Screenshot 2024-04-04 120445](https://github.com/leokhoa/laragon/assets/144902423/362c4138-b357-42e5-bbf3-d59a682889bb)

This is my first time using Laragon, and when I wanted to connect to the server a problem like this appeared

**Comment by Pen-y-Fan** [comment]
Welcome to Laragon.

Do you have any other programs already running on port 80?

From the Laragon menu > Tools > Netstat

<img width="325" alt="image" src="https://github.com/leokhoa/laragon/assets/40126936/fa04df47-c97a-4554-bbac-f167e203f286">

The report will open in notepad++, look for the item **0.0.0.0:80**:

```txt
Tip: Run Laragon as Administrator when you want to see all processes.
----------------------------------------------------------------------------
  Proto  Local Address          Foreign Address        State           PID     Path
  TCP    0.0.0.0:80             0.0.0.0:0              LISTENING       3188    F:\laragon\bin\apache\httpd-2.4.54-win64-VS16\bin\httpd.exe
```

In my case **httpd** is running, which is Apache.

Stop any programs which are running.

You can also try a restart of your computer, also windows updates can affect Laragon, so try an update.

Let us know how ou get on.

**Reply by PPPPPP130** [reply]
i change the port to 90 but still same even though there's no program in there

**Comment by PPPPPP130** [comment]
Now it can be ran because there is a version of vcruntime that is not compatible with the php version
